force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.js

```diff
@@ -53,6 +54,7 @@ export default class SettingsSaveCanvas extends LightningElement {
                 if (result === true) {
                     // update successful
                     this.switchEditMode(false);
+                    this.showToast("success", "Save Complete", this.labelReference.successMessage);
```

Comment:
This will work for now. It's interesting how slow the delay on the toast is in comparison to the actual save resolving. I'm not too worried about race conditions on async refresh apex at this juncture.